daprdocs/static/docs/open-telemetry-collector/collector-config.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: appconfig
+  namespace: default
+spec:
+  tracing:
+    samplingRate: "1"
+    zipkin:
+      endpointAddress: "http://otel-collector.default.svc.cluster.local:9411/api/v2/spans"
```

Comment:
This might be too late for this but why not simply `endpoint`?

Reply:
That's a good call. It's not too late.

cc @msfussell I like this suggestion here ^^^. We can still rename things now. WDYT

Reply:
I prefer to keep endpointAddress, since this is descriptive and there is no real need to shorten this.